packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -354,31 +355,29 @@ function createMergedTransformStream(
   })
 }
 
+const CLOSE_TAG = '</body></html>'
```

Comment:
The same close tag is used in all cases, and it has `ENCODED_TAGS.CLOSED.BODY_AND_HTML` precomputed.